<launch>


  <node type="laser_scan_assembler" pkg="laser_assembler"
        name="tilt_scan_assembler">
    <remap from="scan" to="tilt_scan"/>
    <param name="max_scans" type="int" value="400" />
    <param name="fixed_frame" type="string" value="/map" />
    <param name="ignore_laser_skew" type="bool" value="true" />
  </node>

  <node type="point_cloud_srv" pkg="nbv_utilities" 
        name="point_cloud_srv">
  </node>

  <node type="point_cloud" pkg="nbv_utilities" 
        name="point_cloud">
  </node>

</launch>